### ISSUE TITLE
Possibly malformed REQUEST-922-MULTIPART-ATTACK.conf causes Apache config test to fail

### ISSUE THREAD

**Issue by modmachinery** [coreruleset/coreruleset#4229]
### Describe the bug

Apache fails if `REQUEST-922-MULTIPART-ATTACK.conf` is present.

### Steps to reproduce

With the coreruleset in place:
```bash
a2enmod security2
apachectl -t
```
```console
AH00526: Syntax error on line 45 of /etc/modsecurity/coreruleset/rules/REQUEST-922-MULTIPART-ATTACK.conf:
Error creating rule: Unknown variable: &MULTIPART_PART_HEADERS
Action '-t' failed.
```

### Expected behaviour

```bash
apachectl -t
Syntax OK
```

### Actual behaviour

Apache will fail a configuration test and/or fail if this file is loaded.

### Additional context

The latest rules were pulled yesterday, Apache failed upon the next automatic reload.

### Your Environment

* CRS version (e.g., v3.2.0): **4.17.1**
* Paranoia level setting: 
* ModSecurity version (e.g., 2.9.3): **2.9.5-1**
* Web Server and version (e.g., apache 2.4.41): **Apache/2.4.52 (Ubuntu)**
* Operating System and version: **Ubuntu 22.04.5 LTS**

**Comment by azurit** [comment]
Hi @modmachinery,

if you are using Ubuntu Jammy, then you are probably running Modsecurity version 2.9.5. Variable `MULTIPART_PART_HEADERS`, which is `REQUEST-922-MULTIPART-ATTACK.conf` using, was added into Modsecurity version 2.9.6. Either upgrade Modsecurity to, at least, 2.9.6 or remove file `REQUEST-922-MULTIPART-ATTACK.conf` from Apache configuration.

**Comment by airween** [comment]
@modmachinery,

if you insist to Ubuntu Jammy, you should try [Digitalwave's](https://modsecurity.digitalwave.hu/) unofficial repository. That provides the latest version of ModSecurity.

**Comment by modmachinery** [comment]
We certainly don't insist on continuing to use Jammy, we're prepping to migrate to 24.04 ASAP. @airween thanks for the tip on the repo, we'll consider this.

For now we've deleted the file from production. Thanks @azurit!
